test/xbmc.py

```diff
@@ -64,8 +64,10 @@ def translatePath(path):
     ''' A stub implementation of the xbmc translatePath() function '''
     if path.startswith('special://home'):
         return path.replace('special://home', os.path.join(os.getcwd(), 'test/'))
+    if path.startswith('special://profile/addon_data/script.module.inputstreamhelper/temp'):
```

Comment:
I think it is unfortunate that there is no special path for an addon's userdata directory, something like: `special://addon_data/temp`. But I guess it should have been a method for `xbmcaddon` instead :-/

This ought to be a regular expression so that it works for other add-ons as well. But no worries, I will take care of it when backporting.

Reply:
Evaluating this again, I really think we should be using `special://masterprofile` instead by default. We probably don't want to make this profile-specific and keep it irrelative to the actual profile used.